docs/administration/sbom-support/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: SBOM Generation Capabilities
+weight: 43
+---
+
+In today's software development landscape,
```

Comment:
it doesn't need to introduce too much background on SBOM as we've already mentioned this in the design document. 

What we need is just to give a tutorial for the end users.